aws_jobs/cyhy-data-extract.py

```diff
@@ -194,12 +217,13 @@ def main():
         start_of_data_collection = now + relativedelta(days=-1, hour=0, minute=0, second=0, microsecond=0)
         end_of_data_collection = start_of_data_collection + relativedelta(days=1)
 
-    if args['--federal']:
-        all_fed_descendants = cyhy_db.RequestDoc.get_all_descendants('FEDERAL')
-        orgs = list(set(all_fed_descendants) - ORGS_EXCLUDED)
-    else:
-        all_orgs = cyhy_db.RequestDoc.get_all_descendants('ROOT')
-        orgs = list(set(all_orgs) - ORGS_EXCLUDED)
+    # Get a list of all non-retired orgs
+    org_filter = {
+            '_id': {'$ne': 'ROOT'},
```

Comment:
`'_id': {'$ne': 'ROOT'}` is unnecessary, since `ROOT` does not have any `host_scans`, `port_scans`, `vuln_scans`, `hosts`, or `tickets` associated with it (same thing goes every other grouping node in our hierarchy, which is why you don't have to worry about excluding them from the list of orgs).